testgrid/config/config.proto

```diff
@@ -52,6 +71,9 @@ message TestGroup {
   // deprecated - always set to true
   bool is_external = 25;
 
+  // Specifies the test name for a test.
+  TestNameConfig test_name_config = 26;
```

Comment:
What you said: right now we ignore it; soon we want to stop ignoring it and use this value.